Server/src/controllers/authenticationController.ts

```diff
@@ -11,6 +12,15 @@ import { IUpdateUserDetail } from './../genericInterfaces/AuthenticationInterfac
  * This controller is responsible for verifying the user request has correct parameters input.
  * After request is verified, the appropriate service can be called to fulfill user signup or login
  */
+
+//helpers
+const passwordSchema = Yup.string()
+  .required()
+  .matches(new RegExp(/^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[#$@!%&*?])[A-Za-z\d#$@!%&*?]{8,30}$/), "Password must contain in between 8 to 30 characters, one uppercase, one number and one special case character")
```

Comment:
Similar as above. I think you can make use of the schema you declared

Reply:
fixed on latest commits